specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v2.0/LUIS-Runtime.json

```diff
@@ -5,11 +5,52 @@
     "version": "2.0"
   },
   "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}/luis/v2.0",
-    "useSchemePrefix": false,
+    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.{AzureCloud}/luis/v2.0",
```

Comment:
This change will prevent the users from using customized domain/containers.
Please revert this change.

If you believe this change is a must, we can kickstart an email thread to discuss abt this.

Reply:
@diberry has this been addressed?

Reply:
@nebadr @AmirGeorge Is this something you can address? I don't think I introduced this issue.

Reply:
@dsgouda @diberry this change has been reverted in the second commit.

Reply:
@nebadr - should I close this PR?

Reply:
@diberry We want to merge this PR. I meant the change has been reverted here in this PR but in the second commit (this change appears to be outdated).

Reply:
@nebadr Do I need to pull something into this PR to fix or it is good to go as it is now?

Reply:
@diberry No, it is good. Thanks.